<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatasetOverviewWidget</class>
 <widget class="QWidget" name="DatasetOverviewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>777</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dataset Overview</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>2</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>5</number>
     </property>
     <widget class="QWidget" name="infoContainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="TagFilterWidget" name="dbSearchWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="infoSplitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <widget class="QGroupBox" name="metadataGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QFormLayout" name="metadataLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="fileNameLabel">
             <property name="text">
              <string>ファイル名:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="fileNameValueLabel"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="imagePathLabel">
             <property name="text">
              <string>画像パス:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="imagePathValueLabel"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="extensionLabel">
             <property name="text">
              <string>拡張子:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="extensionValueLabel"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="formatLabel">
             <property name="text">
              <string>フォーマット:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="formatValueLabel"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="modeLabel">
             <property name="text">
              <string>モード:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="modeValueLabel"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="alphaChannelLabel">
             <property name="text">
              <string>アルファチャンネル:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="alphaChannelValueLabel"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="resolutionLabel">
             <property name="text">
              <string>解像度:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="resolutionValueLabel"/>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="aspectRatioLabel">
             <property name="text">
              <string>アスペクト比:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLabel" name="aspectRatioValueLabel"/>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="annotationGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QTextEdit" name="tagsTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QTextEdit" name="captionTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="tagsLabel">
             <property name="text">
              <string>タグ:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="captionLabel">
             <property name="text">
              <string>キャプション:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="imageContainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="ImagePreviewWidget" name="ImagePreview" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ThumbnailSelectorWidget" name="thumbnailSelector" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImagePreviewWidget</class>
   <extends>QWidget</extends>
   <header location="global">ImagePreviewWidget</header>
   <container>1</container>
   <slots>
    <slot>load_image()</slot>
    <slot>update_model_options()</slot>
    <slot>send_vision_prompt()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>ThumbnailSelectorWidget</class>
   <extends>QWidget</extends>
   <header location="global">ThumbnailSelectorWidget</header>
   <container>1</container>
   <slots>
    <signal>on_thumbnail_clicked()</signal>
    <signal>imageSelected()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>TagFilterWidget</class>
   <extends>QWidget</extends>
   <header location="global">TagFilterWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
